packages/lesswrong/lib/collections/posts/views.ts

```diff
@@ -632,25 +635,51 @@ Posts.addView("scheduled", (terms: PostsViewTerms) => ({
  * @summary Draft view
  */
 Posts.addView("drafts", (terms: PostsViewTerms) => {
-  let query = {
+  let query: any = {
     selector: {
       userId: viewFieldAllowAny,
       $or: [{userId: terms.userId}, {shareWithUsers: terms.userId}],
       draft: true,
-      deletedDraft: false,
       hideAuthor: false,
       unlisted: null,
       groupId: null, // TODO: fix vulcan so it doesn't do deep merges on viewFieldAllowAny
       authorIsUnreviewed: viewFieldAllowAny,
       hiddenRelatedQuestion: viewFieldAllowAny,
+      deletedDraft: false,
     },
     options: {
       sort: {}
     }
   }
-  switch (terms.sortDrafts) {
-    case 'wordCount': {
-      query.options.sort = {wordCount: -1, modifiedAt: -1, createdAt: -1}
+  
+  if (terms.includeDraftEvents) {
```

Comment:
It's generally better to include this sort of thing in the initializer if you can, rather than mutating the object you created after the fact. (Heuristic: minimize mutation in general.)